CONTRIBUTING.md

```diff
@@ -106,7 +112,21 @@ PRs must have a category prefix that is based on the type of changes being made
 [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
 Additionally, **each PR should only address a single issue**.
 
-NOTE: when merging, GitHub will squash commits and rebase on top of the main.
+**Note:** When merging, GitHub will squash commits and rebase on top of the main.
+
+#### Large / complex contributions
+
+For large/complex changes, it is **absolutely** required
+to have up-front discussion in issues/GitHub discussions. 
+In some cases, it would be preferable to even have 
+synchronous design sessions to socialize a collective understanding 
+of the nature of incoming changes prior to the work even starting.
+
+It needs to be feasible for our team to review large/complex changes.
+This means that changes either need to be submitted in multiple small PRs 
```

Comment:
Should the multiple small PRs target a feature branch? otherwise we might end up with half-done changes on main, if someone starts contributing for a feature, has parts merged in main, but then stops working on it

Reply:
That's a good point. I'll update.